app/models/review_assignment.rb

```diff
@@ -189,9 +191,10 @@ def contributor_to_review(reviewer, topic)
     raise "You have already reviewed all submissions for this #{work}." if contributor_set.empty?
 
     # Reduce to the contributors with the least number of reviews ("responses") received
+    # Reviewers should be able to review even if he completed all the min reviewed submissions.
     min_contributor = contributor_set.min_by {|a| a.responses.count }
     min_reviews = min_contributor.responses.count
-    contributor_set.reject! {|contributor| contributor.responses.count > min_reviews }
+    contributor_set.reject! {|contributor| contributor.responses.count > min_reviews + review_topic_threshold}
```

Comment:
Consider another case:
If B is a reviewer. Currently the review strategy is each artifact can get 10 reviews and threshold is 2. If the work got minimum reviews have received 2 reviews and B happens to have reviewed all the work which have been reviewed <=4 times (those are the ones considered as reviewable), then if B still wants to review more, B should be able to request works with 5 reviews responses to review.